### DISCUSSION TITLE
Add calendar widget

### DISCUSSION THREAD

**Discussion by Zorg77127** [sqlpage/SQLPage#474] [Ideas]
Hello community,

I don't know if it already exists, maybe yes, and sorry in advance if it does, but I wanted to know if it was possible to create a "calendar widget", for example to select dates / hours to store them in the sqlite db to manage a schedule / planning meeting like  >> [this](https://support.weekcal.com/hc/en-us/articles/8801925315611-WeekCal-Home-Screen-Widgets) << for example.
Thanks in advance and thanks for improving SqlPage !

**Comment by lovasoa** [comment]
Hello and walcome to SQLPage ! 
  Indeed a calendar [has been asked before](https://github.com/lovasoa/SQLpage/issues/104), and would be a nice improvement. Can you tell us more about what you are building ?

**Comment by Zorg77127** [comment]
I'm thinking about developing software for my daughter who is aesthetician, which allows her to take appointment with customers, and maybe going further by doing accounting and bookkeeping.

**Reply by lovasoa** [reply]
That sounds great ! Don't hesitate to share your progress here and to let us know how we can help !

While waiting for a proper calendar component, you may want to use the [card](https://sql.ophir.dev/documentation.sql?component=card#component) component to represent week days.

**Reply by Zorg77127** [reply]
Why not, let's try this way, thanks for this reflection.
